src/erlazure_insights.erl

```diff
@@ -73,9 +73,12 @@ http_request(SubscriptionId, Filter, Select, Config) ->
     URL = "https://management.azure.com/subscriptions/" ++
     SubscriptionId ++
     "/providers/microsoft.insights/eventtypes/management/values?" ++ "api-version=2014-04-01" ++ 
-    "&%24filter=" ++ Filter ++
-    "&%24select=" ++ Select,
-    request_api(URL, Config).
+    "&%24filter=" ++ Filter,
```

Comment:
@lilrooness it actually looks like we do not need to encode this one, at least via curl I didn't:

```
curl -H  "Accept:application/json"  -H "Authorization: Bearer $TOKEN" 'https://management.azure.com/subscriptions/SUB_ID/providers/microsoft.insights/eventtypes/management/values?api-version=2014-04-01&$filter=eventTimestamp%20ge%20%272015-11-29T22:00:37Z%27' | jq .
```

this ^^ request worked for me